src/lib.rs

```diff
@@ -285,6 +299,50 @@ impl Hasher for XxHash {
     }
 }
 
+impl Digest for XxHash {
+    type OutputSize = U8;
+
+    fn new() -> Self {
+        Self::default()
+    }
+
+    fn input<B: AsRef<[u8]>>(&mut self, data: B) {
+        self.write(data.as_ref());
+    }
+
+    fn chain<B: AsRef<[u8]>>(mut self, data: B) -> Self
+    where
+        Self: Sized,
+    {
+        self.input(data);
+        self
+    }
+
+    fn result(self) -> GenericArray<u8, Self::OutputSize> {
+        let mut buf = [0u8; 8];
+        BigEndian::write_u64(&mut buf, self.finish());
```

Comment:
Can this use [`to_be_bytes`](https://doc.rust-lang.org/std/primitive.u64.html#method.to_be_bytes) instead of relying on byteorder?

Reply:
🤦‍♂ I knew it wasn't nice to be pulling in ByteOrder for a single use, I'll change it now :)